source/guide_healthchecks.rst

```diff
@@ -0,0 +1,742 @@
+   Downloading mysql-0.0.3-py3-none-any.whl.metadata (746 bytes)
+ Collecting mysqlclient (from mysql)
+   Using cached mysqlclient-2.2.4-cp311-cp311-linux_x86_64.whl
+ Downloading mysql-0.0.3-py3-none-any.whl (1.2 kB)
+ Installing collected packages: mysqlclient, mysql
+ Successfully installed mysql-0.0.3 mysqlclient-2.2.4
+
+ [notice] A new release of pip is available: 23.2.1 -> 24.0
+ [notice] To update, run: pip install --upgrade pip
+ [isabell@stardust ~]$ pip3.11 install gunicorn
+ Collecting gunicorn
+   Obtaining dependency information for gunicorn from https://files.pythonhosted.org/packages/0e/2a/c3a878eccb100ccddf45c50b6b8db8cf3301a6adede6e31d48e8531cab13/gunicorn-21.2.0-py3-none-any.whl.metadata
+   Using cached gunicorn-21.2.0-py3-none-any.whl.metadata (4.1 kB)
+ Collecting packaging (from gunicorn)
+   Obtaining dependency information for packaging from https://files.pythonhosted.org/packages/49/df/1fceb2f8900f8639e278b056416d49134fb8d84c5942ffaa01ad34782422/packaging-24.0-py3-none-any.whl.metadata
+   Downloading packaging-24.0-py3-none-any.whl.metadata (3.2 kB)
+ Using cached gunicorn-21.2.0-py3-none-any.whl (80 kB)
+ Downloading packaging-24.0-py3-none-any.whl (53 kB)
+    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 53.5/53.5 kB 2.9 MB/s eta 0:00:00
+ Installing collected packages: packaging, gunicorn
+ Successfully installed gunicorn-21.2.0 packaging-24.0
+
+ [notice] A new release of pip is available: 23.2.1 -> 24.0
+ [notice] To update, run: pip install --upgrade pip
+ [isabell@stardust ~]$
+
+Configuration
+=============
+
+Configure the app
+------------------------
+
+Edit the file ``./hc/local_settings.py`` and set your host specific settings:
+
+.. code-block:: python
+
+ DEBUG = False
+
+ SITE_ROOT = 'https://isabell.uber.space'
+ SITE_NAME = 'My Monitoring Project'
+ PING_ENDPOINT='https://isabell.uber.space/ping/'
+ DEFAULT_FROM_EMAIL = 'healthchecks@isabell.uber.space'
+ ALLOWED_HOSTS=['isabell.uber.space']
+ CSRF_TRUSTED_ORIGINS=['https://isabell.uber.space']
+ SECRET_KEY = '<PLACE A RANDOM STRING HERE>'
+ REGISTRATION_OPEN = False # True if you'd like to allow user registrations of anyone from the internet
+
+ DATABASES = {
+     'default': {
+         'ENGINE': 'django.db.backends.mysql',
+         'HOST': 'localhost',
+         'PORT': '3306',
+         'NAME': 'isabell_healthchecks',
+         'USER': 'isabell',
+         'PASSWORD': 'MySuperSecretPassword',
+         'TEST': {'CHARSET': 'UTF8'}
+     }
+ }
+
+ # Email
+ EMAIL_HOST = 'stardust.uberspace.de'
+ EMAIL_PORT = 465
+ EMAIL_HOST_USER = 'healthchecks@isabell.uber.space'
+ EMAIL_HOST_PASSWORD = 'MySuperSecretPassword'
+ EMAIL_USE_TLS = False
+ EMAIL_USE_SSL = True
+
+
+
+Initialize the database
+-----------------
+
+Once we set the correct settings we can initialize the database:
+
+::
+
+ (venv) [isabell@stardust ~]$ python3.11 ./manage.py migrate
```

Comment:
Here (and also in some of the following listings) I assume that the user is still in the active virtualenv. Do you want me to repeat the `source venv/bin/activate` command before each of the commands requiring the virtualenv?

Reply:
Good question. If this was my guide I would provide a note before the first use of the `venv` with something like that:

```
.. note:: All following commands require the virtualenv, you can activate it with ``source venv/bin/activate`` in a new SSH-session
```